test/azure/generated/AzureCompositeModelClient/azureCompositeModel.ts

```diff
@@ -29,10 +30,17 @@ class AzureCompositeModel extends AzureCompositeModelContext {
 
   /**
    * Initializes a new instance of the AzureCompositeModel class.
-   * @param credentials Credentials needed for the client to connect to Azure.
+   * @param credentials Credentials needed for the client to connect to Azure. The simplest
+   * TokenCredential credential can be obtained as follows:
+   * ```js
+   * const { DefaultAzureCredential } = require("@azure/identity");
+   * const credential = new DefaultAzureCredential();
+   * ```
+   * For more information about these credentials, see {@link
+   * https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity#authenticating-with-the-defaultazurecredential}
```

Comment:
Does this comment render correctly in VSCode tooltip>? I wonder if the docs tools would have any issues because `{@link` and link are not in the same line

Reply:
It would be odd that the constructor supports two credential types, but our docs for the parameter talks only about the second one. Moreover, we don't want to alarm users into thinking that the package they are currently using is not supported anymore. Because it is not easy to determine just looking at the type `msRest.ServiceClientCredentials` that their old code would still work.

I would recommend going with something like this:

> Credentials needed to authenticate the client using Azure Active Directory. 
Credentials implementing the `TokenCredential` interface from the @azure/identity package are recommended. 
Credentials implementing the `ServiceClientCredentials` interface from the older packages @azure/ms-rest-nodeauth and @azure/ms-rest-browserauth are also supported.